src/Eyeball.jl

```diff
@@ -467,6 +469,10 @@ function getoptions(x::AbstractSet{T}) where T
     return zip(Iterators.repeated(Symbol("")), x)
 end
 
+function __init__()
+    @require MAT="23992714-dd62-5051-b70f-ba57cb901cac" getoptions(x::MAT.MAT_HDF5.MatlabHDF5File) = zip(keys(x), (read(x,k) for k in keys(x)))
```

Comment:
Use of `read` is an issue (if I'm interpreting it right). If someone runs `eye`, and there's an H5 file loaded that has huge arrays, it could take forever and/or crash something.

Is there a way to read a summary of that or the read the first part of that rather than reading the whole thing?